.gitignore

```diff
@@ -27,5 +27,8 @@ Thumbs.db
 # SMP artifacts from testing
 /*.smp
 
-# Published directory for type definitions
+# Published directory for npm
 dist/
+
+# tsup
```

Comment:
Nitpick: what does this comment add?

Reply:
it's meant to document the addition of `.tsup/`, which I believe is an intermediate artifact of tsup - can make that clearer

Reply:
ah actually this directory is only created if using the `experimentalDts` option: https://tsup.egoist.dev/#generate-declaration-file

Will just remove in that case

Reply:
Done via 6f0d4590dd3bf1fcdb6776ff4aac9cd00301541a